content/fr/blog/2021-04-27-cant-read-wont-buy.md

```diff
@@ -1,92 +1,93 @@
+> L'hypothèse selon laquelle les acheteurs potentiels "parlent probablement anglais" conduit à une localisation inadéquate, ce qui va à l'encontre de l'intuition selon laquelle il est peu probable que les gens achètent des produits qu'ils ne comprennent pas ou qui ne les attirent pas. (CRWB 2006)
 
-Is the problem just about language? No, it's also about meeting consumers where they are in terms of the ways in which they are used to spending:
+Le problème est-il uniquement lié à la langue ? Non, il s'agit également d'aller à la rencontre des consommateurs là où ils se trouvent, c'est-à-dire là où ils ont l'habitude de dépenser :
 
-> Even with information available in the local language, the inability to use their own credit cards or currency stymies many international buyers. (CRWB 2006)
+> Même si les informations sont disponibles dans la langue locale, l'impossibilité d'utiliser leurs propres cartes de crédit ou leur propre monnaie freine de nombreux acheteurs internationaux. (CRWB 2006)
 
-The relative cost of the good or service is also significant, and it's important to note that determining what cost should be &mdash; e.g taking into account the local economic circumstances &mdash; is also a crucial component of effective localization:
+Le coût relatif du bien ou du service est également significatif, et il est important de noter que la détermination de ce que devrait être ce coût, par exemple en tenant compte des circonstances économiques locales, est également un élément crucial d'une localisation efficace :
 
-> The more valuable an item, the more likely it is that someone will want to read about the product and buy it in their own language. (CRWB 2006)
+> Plus un article a de la valeur, plus il est probable que quelqu'un voudra lire des informations sur le produit et l'acheter dans sa propre langue. (CRWB 2006)
 
-However, *language* is the primary blocker to successful internationalization; while 67.4% of the non-native English speakers polled said they regularly visited English-language websites, 74.5% rarely or never go on to make purchases there.
+Cependant, la *langue* est le principal obstacle à une internationalisation réussie ; tandis que 67,4 % des personnes interrogées dont l'anglais n'est pas la langue natale, ont déclaré qu'elles visitaient régulièrement des sites Web en langue anglaise, 74,5 % d'entre elles n'y effectuent que rarement ou jamais d'achats.
 
-![Visiting English-Language Websites](/images/blog/2021-04-27-non-native-visits.png)
+![Visiter des sites Web anglophones](/images/blog/2021-04-27-non-native-visits.png)
 
-![Buying on English-Language Websites](/images/blog/2021-04-27-cant-read-wont-buy.png)
+![Achats sur les sites Web anglophones](/images/blog/2021-04-27-cant-read-wont-buy.png)
```

Comment:
I'm guessing that capturing the links for translation was tricky; thank you for figuring that out as well!